utils/waitUntil.ts

```diff
@@ -0,0 +1,7 @@
+export default async function waitUntil(time: number = 0): Promise<"timeout"> {
```

Comment:
do we want to have a default as some seconds over 0?

Reply:
true! addressed in addressed in https://github.com/cennznet/app-hub/pull/139